Src/Microsoft.Dynamic/Generation/ILGen.cs

```diff
@@ -251,7 +251,7 @@ public class ILGen {
             _ilg.EmitCall(opcode, methodInfo, optionalParameterTypes);
         }
 
-#if !NETCOREAPP2_0 || NETCOREAPP2_1
+#if (!NETCOREAPP2_0 && !NETSTANDARD2_0) || NETCOREAPP2_1
```

Comment:
Nit: I'm not sure why we had the `|| NETCOREAPP2_1` here. Doesn't do anything and can be removed.

Reply:
Done